admin/pre-reg/database.py

```diff
@@ -0,0 +1,107 @@
+import httplib as http
+
+from modularodm import Q
+
+from website.project.model import MetaSchema, DraftRegistration, DraftRegistrationApproval
+from framework.mongo.utils import get_or_http_error
+from framework.auth.core import User
+from framework.auth import Auth
+from website.project.metadata.utils import serialize_meta_schema
+from website.app import do_set_backends, init_addons
+from website import settings as osf_settings
+
+import utils
+
+# [lauren] I think this is done in manage-admin.py now
+init_addons(osf_settings, routes=False)
+do_set_backends(osf_settings)
+
+# TODO[lauren]: remove once all auth is taken out of project
+adminUser = User.load('dsmpw')
+
+
+def get_all_drafts():
+    """Retrieves all submitted prereg drafts from OSF db
+    :return: Dict of submitted prereg drafts
+    """
+    # TODO[lauren]: add query parameters to only retrieve submitted drafts
+    # they will have an approval associated with them
+    all_drafts = DraftRegistration.find()
+    # TODO[lauren]: change to current user
+    auth = Auth(adminUser)
+
+    serialized_drafts = {
+        'drafts': [utils.serialize_draft_registration(d, auth) for d in all_drafts]
```

Comment:
Even though it isn't DRY it might be better to create a new serialization in order not to use auth.